<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Console</class>
 <widget class="QWidget" name="Console">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1460</width>
    <height>412</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>350</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Console</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="rightMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QGroupBox" name="LoggingLevel">
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>0</height>
        </size>
       </property>
       <property name="title">
        <string>Logging</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
       <widget class="QWidget" name="verticalLayoutWidget">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>131</width>
          <height>251</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QCheckBox" name="trace_check">
           <property name="text">
            <string>Trace</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="debug_check">
           <property name="text">
            <string>Debug</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="info_check">
           <property name="text">
            <string>Info</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="warn_check">
           <property name="text">
            <string>Warn</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="error_check">
           <property name="text">
            <string>Error</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="fatal_check">
           <property name="text">
            <string>Fatal</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="log_file">
           <property name="layoutDirection">
            <enum>Qt::LeftToRight</enum>
           </property>
           <property name="text">
            <string>Log to File</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="clear_button">
           <property name="text">
            <string>Clear</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QPlainTextEdit" name="log_messages"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
